t/execution-variables.t

```diff
@@ -154,6 +154,24 @@ subtest 'Handles objects and nullability', sub {
       );
     };
 
+    subtest 'errors on incorrect query input', sub {
```

Comment:
This is a great catch. Could you put the test in `t/perl.t` unless you're copying a test from the JS original? That's where I put all the implementation-specific ones, which this certainly is.